pom.xml

```diff
@@ -43,6 +43,9 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
+        <!-- bump version when performing releases -->
+        <geomesa.release.version>1.1.0-rc.6</geomesa.release.version>
```

Comment:
Tried adding it and I can only get "1.1.0" to be filtered into the pom, but not "-rc.6".